### PR TITLE
man: add Russian translation

### PR DESCRIPTION
**Pull request by egor1wtf**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by egor1wtf** [comment]
Hello maintainers. I'm sorry, I made a lot of mistakes in the past PR. I had to create a new one.

Please check if everything is correct. Thank you and have a nice day!

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and Thanks for your contribution.

File: pages.ru/linux/man.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+> Утилита просмотра справочных страницs.
+> Больше информации: <https://manned.org/man>.
+
+- Показать справочную страницу для команды:
+
+`man {{команда}}`
+
+- Показать справочную страницу пакета макросов команды из раздела 7:
+
+`man {{7}} {{команда}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/linux/man.md:12]:
```suggestion
- Показать справочную страницу пакета макросов команды из раздела:

`man {{1..9}} {{команда}}`
```

Mandatory fix (error source: original page): [unexpected `7` number, range construct expected](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#range-syntax).

File: pages.ru/linux/man.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`man --path`
+
+- Отобразить расположение справочной страницы, а не саму справочную страницу:
+
+`man --where {{команда}}`
+
+- Отобразить справочную страницу с использованием определённой локали:
+
+`man --locale={{locale}} {{команда}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/linux/man.md:28]:
```suggestion
- Отобразить справочную страницу с использованием определённой локали:

`man --locale={{строка}} {{команда}}`
```

Optional fix (error source: original page): unexpected `locale` term, `string` expected.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
@egor1wtf, notify me please when this PR can be merged (or anybody else can merge it). :)

File: pages.ru/linux/man.md (hunk 2 above)
Comment (egor1wtf) [reply pages.ru/linux/man.md:28]: @EmilySeville7cfg 

Thanks you for the suggestions!

I disagree about the change from argument: --locale={{locale}} to --locale={{строка}}.

A word "строка" translated in English means "string". It's not entirely clear what "string" to enter for the user. I propose to use the transcription of the English word "locale" into Russian "локаль".

The final version will look like this:

```suggestion
- Отобразить справочную страницу с использованием определённой локали:

`man --locale={{локаль}} {{команда}}`
```

I look forward to your thoughts and suggestions. Thank you

Comment (EmilySeville7cfg) [reply pages.ru/linux/man.md:28]: Your idea is neat, I like it. Let's use what u suggested as there is no word duplication.

**Comment by egor1wtf** [comment]
@EmilySeville7cfg 

Thank you for your help! I'm all set, can you please approve the PR?

**Review by kbdharun — APPROVED** [review]

**Pull request merged**